src/processor/template_processor/base/base_template_constatns.py

```diff
@@ -10,6 +11,7 @@
     "arm" : AzureTemplateProcessor,
     "deploymentmanager" : GoogleTemplateProcessor,
     "terraform" : TerraformTemplateProcessor,
+    "kubernetesObjectFiles" : KubernetesTemplateProcessor
```

Comment:
Add the comma(",") at the end.

Reply:
I added comma for resolve merging conflicts purpose